transform-plugins/widgets/ValueMapper-transform.json

```diff
@@ -0,0 +1,17 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "properties": [
+        {
+          "widget-type": "textbox",
```

Comment:
As per Sree's suggestion, Using text field as temporary widget. However , tried using json widget but got some issue as it was not showing the json object I have entered in the widget after I saved the pipeline. Not sure if I have used json widget incorrectly. Thus switched to text field widget and passing json string as input, for temporary testing purpose.